src/lib.rs

```diff
@@ -93,175 +55,23 @@ impl fmt::Display for UserLed {
     }
 }
 
-/// Set of available options to select PWM's channel.
-#[derive(Debug, Clone, Copy, PartialEq)]
-pub enum PwmChannel {
-    Ch1,
-    Ch2,
-    Ch3,
-    Ch4,
-    Ch5,
-    Ch6,
-    Ch7,
-    Ch8,
-    Ch9,
-    Ch10,
-    Ch11,
-    Ch12,
-    Ch13,
-    Ch14,
-    Ch15,
-    Ch16,
-    All,
-}
-
-/// The `AxisData` struct encapsulate values for the x, y, and z axes.
+/// The `AxisData` struct encapsulates values for the x, y, and z axes.
 #[derive(Debug, Default, Clone, Copy)]
 pub struct AxisData {
     pub x: f32,
     pub y: f32,
     pub z: f32,
 }
 
-/// Encapsulates the value of ADC's four channels.
-#[derive(Debug, Default, Clone, Copy)]
-pub struct ADCData {
-    pub channel: [f32; 4],
-}
-
-/// Encapsulates the value of all sensors on the board.
```

Comment:
This commit isn't changing only ADCData to vector, but also removing SensorData. SensorData and read_all will be droped out in newer api?

Reply:
Fixed